app/views/submissions/new.html.haml

```diff
@@ -1,10 +1,17 @@
-%h1.heading-xlarge
-  Upload submission for
-  = @task.framework.short_name
+%h1.govuk-heading-xl
+  Upload a file
```

Comment:
Was wondering what the reasoning behind removing mentions of the framework name from the upload page? It feels useful to have a reminder on this page about which framework they should be submitting for..?

Reply:
There is a component to go back in I'm currently calling 'task signpost' that indicates the task and frame work name throughout. I wanted to check the backend supplies this content to every page in the journey before I add it in another PR.

Reply:
:+1: